src/microbit/pxt_send_en_hemmelig_beskjed/send_en_hemmelig_beskjed.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "PXT: Send en hemmelig beskjed"
+author: Helene Isnes
+language: nb
+---
+
+
+# Introduksjon {.intro}
+
+I denne oppgaven skal du sende og motta hemmelige beskjeder til/fra en annen sin micro:bit ved å bruke radiosignaler. Du trenger derfor en å utveksle hemmelige beskjeder med! Finn noen som har lyst til å gjøre denne oppgaven samtidig med deg.
+
+
+# Steg 1: Sett opp en radiogruppe {.activity}
+
+## Sjekkliste {.check}
+
+- [ ] Finn en `ved start`-kloss (den ligger allerede i kodefeltet eller du finner den nederst i `Basis`). 
+
+- [ ] Gå til `Radio` og klikk på `radio sett gruppe`-klossen. 
+
+- [ ] Velg dere et gruppenummer mellom 0 og 255 som dere tror ingen andre har. Alle micro:biter som er i samme gruppe kan kommunisere, og du vil ikke at noen andre skal høre den hemmelige beskjeden!
+
+![Bilde som viser radio sett gruppe til 42 kloss inni ved start klossen](radiogruppe.png)
+
+
+# Steg 2: Lag hemmelig beskjed {.activity}
+
+## Sjekkliste {.check}
+
+- [ ] Gå til `Inndata` og finn klossen `når knapp A trykkes`. Vi bruker denne fordi vi vil sende den hemmelige beskjeden når vi trykker på A. 
+
+- [ ] Finn klossen `radio send tekst` i `Radio`. Legg den inn i `når knapp A trykkes` og skriv inn den hemmelige beskjeden du vil sende. 
+
+![Bilde som viser at når A trykkes så skal radio sende teksten "Hemmelig beskjed"](hemmelig_beskjed.png)
+
+### OBS! {.protip}
+
+Micro:biten klarer ikke å vise Æ, Ø eller Å. Bruk heller AE, OE og AA. Beskjeden må heller ikke være for lang. 
```

Comment:
vet vi hvor lang beskjeden kan være? hva skjer om den er for lang?

Reply:
micro:biten klarer bare å sende noen få ord (det meste jeg har fått sendt er 19 tegn). Hvis beskjeden er for lang sender den ikke hele, men bare de første ordene
Kan det gå an å skrive: "Beskjeden burde heller ikke være for lang siden micro:biten bare klarer å sende noen få ord av gangen", eller noe i den duren?

Reply:
skjønner. det høres ut som en bedre ordlyd ja :)